conf/local.conf.sample

```diff
@@ -280,3 +280,14 @@ BB_DISKMON_DIRS = "\
 # track the version of this file when it was generated. This can safely be ignored if
 # this doesn't mean anything to you.
 CONF_VERSION = "1"
+
+# WIC image type support
+IMAGE_FSTYPES ?= "tar.bz2 wic.bz2"
+
+# dosfstools is needed to create the boot partition, mtools to copy to it
+IMAGE_DEPENDS_wic_append = " dosfstools-native mtools-native"
+
+# Uncomment one of these for specific SD card sizes (default=4g)
+WKS_FILE ?= "sdimage-bootpart-4g.wks"
+# WKS_FILE ?= "sdimage-bootpart-2g.wks"
+# WKS_FILE ?= "sdimage-bootpart-8g.wks"
```

Comment:
[**@fahadusman**](https://github.com/fahadusman) commented on [Sep 30, 2015, 10:07 AM MST](https://github.com/MentorEmbedded/meta-mentor/pull/567/files#r40821213 "2015-09-30T17:07:36Z - Replied by Github Comment Enhancer"):
> For the machines that require machines-specific changes in wks files, we will use a set of wks files with the same name in a more specific layer but you're right, it won't work with qemu machines.

Overridding the wks without changing its name, but substantially changing its content, is not ideal. If it needs its sd card done differently, we'll need new wks files with appropriate names and WKS_FILE set appropriately in the machine .conf.